### ISSUE TITLE
Example does not work

### ISSUE THREAD

**Issue by hanusto** [eventuate-tram/eventuate-tram-core-examples-basic#7]
I am trying to use this example with docker-compose up. Everything seems to be ok, but, when I add record into message table (manually via dbeaver client) nothing happens. 

What is wrong? My expectation is that when something is inserted into MESSAGE table, new kafka event is emitted.

**Comment by cer** [comment]
What's the output of `docker-compose ps` and `docker-compose logs cdcservice`?
The CDC service needs to be running in order to publish messages inserted into the `MESSAGE` table.

**Comment by cer** [comment]
Also, what's the value of `DOCKER_HOST_IP`?

**Comment by hanusto** [comment]
docker-compose logs cdcservice -> everything looks like good. CDC service is running.

WARNING: The DOCKER_HOST_IP variable is not set. Defaulting to a blank string.

I am using docker-compose as is.

**Comment by hanusto** [comment]
Hi,currently everything works as expected. Mistake was between pc table and computer :). By the way, is there any documentation of implementation? For example what has to be done if someone want to use it. I am looking for documentation where I found explanation how it works, which Spring Boot properties have to be configured and why and so no.

**Comment by cer** [comment]
@hanusto I expanded the README. Please review.

**Comment by hanusto** [comment]
Yes, it is great. And is there documentation where I can found how can I improve this code for example with AVRO integration? Which class I have to overload, or which configuration I have to change? Maybe I miss architecture overview or detail description how CDC is implemented. For example, now I have not MySQL db, but postgreSQL or MSSQL. So theoretically I can use the same approach but I have to change implementation of CDC service to read tx log for specific DB or use debezium engine, but how...

I can cooperate with you, for my opinion as someone who wants to use it some important notes are missing. I am thinking about using it in our project, but documentation is very high-level. Sorry for my feedback ;)

**Comment by cer** [comment]
Docs definitely need work. Fixing this is a priority.

See this WIP manual on the CDC - http://eventuate.io/docs/manual/eventuate-tram/wip/

The CDC supports MySQL and postgres.

**Comment by hanusto** [comment]
Wow! It is perfect. And why this is not linked in public website? Because it is in work in progress? But it is exactly what I am looking for :) Can I help you?

**Comment by cer** [comment]
@hanusto Sorry for the delayed response. Crazy work schedule. I've revised the docs. It would be great if you could take a look.

**Comment by sanjeevgour** [comment]
@hanusto Hey Tomas - Were you able to figure out how to integrate Avro with this?
